router/xgress_edge_tunnel/tunneler.go

```diff
@@ -138,3 +143,56 @@ func (self *tunneler) Close() error {
 	self.interceptor.Stop()
 	return nil
 }
+
+func (self *tunneler) HandleReconnect() {
+	terminators := self.terminators.Items()
+	for _, terminator := range terminators {
+		terminator.created.Store(false)
+	}
+
+	select {
+	case self.notifyReconnect <- struct{}{}:
+	default:
+	}
+}
+
+func (self *tunneler) ReestablishmentRunner() {
+	for {
+		<-self.notifyReconnect
+		self.ReestablishTerminators()
+	}
+}
+
+func (self *tunneler) ReestablishTerminators() {
+	log := pfxlog.Logger()
+	terminators := self.terminators.Items()
+
+	if len(terminators) > 0 {
+		pfxlog.Logger().Infof("reestablishing %v terminators", len(terminators))
```

Comment:
this really info? it seems more debugish to me. as a consumer of this log, it wouldn't mean much?

Reply:
I'm not fussed either way. I'll switch it to a debug

Reply:
Fixed